README.md

```diff
@@ -1,5 +1,9 @@
-# autoware-documentation
+# Autoware documentation
 
 - This repository contains the [MkDocs](https://www.mkdocs.org/) source files for the [Autoware documentation hosted on GitHub Pages](https://autowarefoundation.github.io/autoware-documentation/main/).
 - Since Autoware is made up of [multiple repositories](https://github.com/autowarefoundation/), a central documentation repository is important to make information accessible from a single place.
 - For more information about Autoware and its related repositories, refer to the [Autoware Foundation's organization profile](https://github.com/autowarefoundation/.github/blob/main/profile/README.md).
+
+## Contributions
+
+To contribute to this repository, see the [corresponding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/documentation-guidelines/).
```

Comment:
I think the URL is not appropriate.

```suggestion
To contribute to this repository, see the [corresponding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
```

Or this one?

```suggestion
To contribute to this repository, see the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
```

Reply:
Or if you intended how to write documents: 

```suggestion
To contribute to this repository, see the [documentation guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/documentation-guidelines/).
```

Reply:
Thank you. I meant only the documentation guidelines.